Firestore/core/include/firebase/firestore/firestore_errors.h

```diff
@@ -23,14 +23,11 @@ namespace firestore {
 /**
  * Error codes used by Cloud Firestore.
  *
- * The codes are in sync with the Firestore iOS client SDK header file
- * FIRFirestoreErrors.h.
+ * The codes are in sync across Firestore SDKs on various platforms.
  */
-enum FirestoreErrorCode {
-  /**
-   * The operation completed successfully. NSError objects will never have a
-   * code with this value.
-   */
+enum Error {
+  /** The operation completed successfully. */
+  // Note: NSError objects will never have a code with this value.
```

Comment:
I think the `NSError` part of the comment shouldn't make it into the public docs.